scrapers/5Kteens.yml

```
@@ -0,0 +1,55 @@
+name: 5Kteens
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 5kteens.com/episodes
+      - 5kporn.com/episodes
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name:
+          selector: //span[starts-with(@class, "color-")]/@class
+          postProcess:
+            - map:
+                color-5KT: 5Kteens
+                color-5KP: 5Kporn
+      Title: //p[@class="trailer-title"]
+      Details: //p[@class=""]
+      Performers:
+        Name:
+          selector: //h5[text()[contains(.,'Starring')]]
+          postProcess:
+            - replace:
+                - regex: Starring:\s([\w\s]+)
+                  with: $1
+      Date:
+        selector: //h5[text()[contains(.,'Published')]]
+        postProcess:
+            - replace:
+                - regex: Published\:\s(\d{4}-\d{2}-\d{2})
+                  with: $1
+      Image:
+        selector: //img[@class="imgres"]/@src
+        postProcess:
+          - replace:
+              - regex: (/episodes/\d+)/.+$
+                with: $1/poster_image/poster_5k.jpg
```

```suggestion
        selector: //img[@class="trailer-poster"]/@src | //script[contains(., "videojs(")]
        postProcess:
          - replace:
              - regex: "^.+poster: \"(.+?)\".+$"
                with: $1
```

This seems like the safest bet, it works for all of these examples (**NSFW**):
- `https://www.5kporn.com/episodes/5KP/1`
- `https://www.5kporn.com/episodes/5KP/85`
- `https://www.5kteens.com/episodes/5KT/1`
- `https://www.5kteens.com/episodes/5KT/32`